jobs/etcd/templates/etcd_bosh_utils.sh.erb

```diff
@@ -27,11 +27,14 @@ node_name="<%= name.gsub('_', '-') %>-<%= spec.index %>"
   client_protocol = p("etcd.require_ssl") ? "https" : "http"
   peer_protocol = p("etcd.peer_require_ssl") ? "https" : "http"
 
+  dns_suffix = p("etcd.dns_suffix")
+  dns_suffix = ".#{dns_suffix}" unless dns_suffix.empty?
```

Comment:
When your DNS search suffix does the right thing, mostly (i.e. `search example.com` in `/etc/resolv.conf`).  In our case, this is a local dev VM thing (that isn't ready for the world, and might cease to exist before that happens) where that's the case.

Not too concerned if we don't want that to be legal (can always change the VM), but it seemed good to correctly support somebody explicitly giving an empty string.